test-cases/gemini/gemini-3h-ics-cdc-with-nemesis.yaml

```diff
@@ -6,6 +6,7 @@ n_monitor_nodes: 1
 instance_type_db: 'i3.large'
 
 user_prefix: 'ics-cdc-gemini-basic-3h'
+ami_db_scylla_user: 'centos'
```

Comment:
this would override both the oracle and scylla under test,

what exactly you are trying to solve here ? which version is being used as the oracle ?

Reply:
Line 34 defines the oracle version as "2020.1.7" which doesn't have user defined by default and used for other gemini jobs with OS Scylla. Also, I tested such config change and it works.

Reply:
or we should revert https://github.com/scylladb/scylla-cluster-tests/commit/29df1ef7f364fd53738d2b0cd14a902fa291fe27

or create a new configuration value for the oracle user

also please don't do PRs directly into branch-2020.1, first go via master.

Reply:
> Line 34 defines the oracle version as "2020.1.7" which doesn't have user defined by default and used for other gemini jobs with OS Scylla. Also, I tested such config change and it works.

anyhow this should be fixed also on master and branch 5.0, and for all the Gemini jobs that has oracle usage.

Reply:
also maybe the oracle should change to 2021.1 ? @aleksbykov any reason why the oracle here is 2020 ?

Reply:
Fix for master branch is here: https://github.com/scylladb/scylla-cluster-tests/pull/4658